js/source/worker_tile.js

```diff
@@ -94,7 +94,7 @@ class WorkerTile {
                 });
 
                 bucket.populate(features, options);
-                featureIndex.bucketLayerIDs[bucket.index] = family.map(getLayerId);
+                featureIndex.bucketLayerIDs[bucket.index] = family.map((l) => l.id);
```

Comment:
This is inside a double-loop and the closure here doesn't need any of the outer scope — I'm not sure V8 optimizes cases like this well enough, so I'd keep it a top-level function.